scripts/plot_v_score.py

```diff
@@ -81,10 +81,12 @@ def get_exact_energies(data):
             ham_attr = row[:2]
             if ham_attr in out:
                 print(f"Warning: Duplicate exact result: {ham_attr}")
-                continue
-
-            energy = row[3]
-            out[ham_attr] = (energy, tag)
+                if energy > row[3]:
```

Comment:
As Alexander added different symmetry sectors in `J1J2/triangular_48_P_0.125.md`, we choose the lowest energy